### PR TITLE
version 0.1

### PR DESCRIPTION
**Pull request by tamershahin**
pls focus on .md file, the .html is autogenerated

### CODE REVIEW

File: API.md [hunk 1]
```diff
@@ -0,0 +1,474 @@
...
+
+```bash
+curl -X POST \
+  'https://routing.staging.simpletrip.de/engine/reservation/v1/reservations?token=300e3921-3c73-415a-8ba5-49d27619d435' \
+  -H 'Accept: application/json' \
+  -H 'Content-Type: application/json' \
+  -H 'accept-encoding: gzip, deflate' \
+  -H 'content-length: 613' \
+  -d '{
+	"optionId": "600124495|734346069",
```
Comment (tbekas) [inline API.md]: The JSON formatting could be improved in this body payload.

File: API.md [hunk 2]
```diff
@@ -0,0 +1,474 @@
...
+
+https://routing.staging.simpletrip.de/engine/reservation
+
+
+### Request example
+In our example, to perform a taxi ride reservation, the parameter list and the payload for the POST call looks like:
+
+```bash
+curl -X POST \
+  'https://routing.staging.simpletrip.de/engine/reservation/v1/reservations?token=300e3921-3c73-415a-8ba5-49d27619d435' \
```
Comment (tbekas) [inline API.md]: Is this a test API-token or some EP related?

File: API.md [hunk 3]
```diff
@@ -0,0 +1,474 @@
...
+The Payment Engine endpoints are reachable with this base url:
+
+https://routing.staging.simpletrip.de/engine/payment
+
+### Request example for GET Payment Options
+```bash
+curl -X GET \
+  'https://routing.staging.simpletrip.de/engine/payment/v1/options?
+  provider=TAXITENDER
+  &token=valid_account_token' \
```
Comment (tbekas) [inline API.md]: Different token than previously. Maybe we should generate the test token for staging only, for people reading this docs.

File: API.md (hunk 1 above)
Comment (tamershahin) [reply API.md]: don't know why.. but the html generating tool wants the formatting being like this to render it properly.. still trying to understand why

File: API.md (hunk 2 above)
Comment (tamershahin) [reply API.md]: I will use `valid_account_token` everywhere

File: API.md (hunk 3 above)
Comment (tamershahin) [reply API.md]: I will use `valid_account_token` everywhere

**Pull request merged**